official/utils/testing/scripts/presubmit.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
It would be nice to provide a command to run this from anywhere in the repo. For instance:
`alias lint_and_test=$(git rev-parse --show-toplevel)/official/utils/testing/scripts/presubmit.sh`
Either in the README or in this file.

Reply:
Have usage example on the top of the script.